dtls/examples/certificates/README.md

```diff
@@ -26,31 +26,31 @@ $ rm "${EXTFILE}" "${SERVER_NAME}.csr" "${CLIENT_NAME}.csr"
 ```
 
 in pion/examples/util/util.go, convert ECPrivateKey to PKCS8PrivateKey
-```
+
+```go
```

Comment:
Hmm, we should remove this code and replace it with a TODO I think.

Reply:
Where would that TODO go and what would it say?

Reply:
Right here and it should probably say `TODO: Port code example from Pion <link>`

Reply:
That piece of code is actually nowhere to be found in [pion/dtls](https://github.com/pion/dtls) at this point.

The closest I could find is [this implementation](https://github.com/pion/dtls/blob/0fbf8d103737d247a36377c739eb34bea6d790a6/examples/util/util.go#L95-L124) of a `func LoadKey(path string) (crypto.PrivateKey, error)`, which was later removed in [this commit](https://github.com/pion/dtls/commit/5c4fb0e2211ae28500bd0a3bf52f2ce5abe66a74) from Apr 26, 2022.

Reply:
Thanks for investigating. Perhaps we just write something to the effect of `TODO: write some example code to convert an EC private key to a PKCS#8 key` 🤷🏼

Reply:
Looks like there exists a crate for this!

[topk8](https://crates.io/crates/topk8) – Convert private keys to PKCS#8 format in pure Rust

Added a rust example based on it!

Reply:
Aight